docs/Downloading-Controlled-Access-Data.md

```diff
@@ -0,0 +1,48 @@
+# Downloading Controlled-Access Data
+
+## [Introduction](introduction)
+A user authentication token (data access token) is required for downloading controlled-access data programmatically from the cBioPortal. See [Obtaining an Authentication Token](#obtaining-an-authentication-token) for instructions on how to download a token and [Using Authentication Tokens](#using-authentication-tokens) for how to use it.
+
+## [Obtaining an Authentication Token](obtaining-an-authentication-token)
+To obtain an authentication token, simply select the `Download Token` option from the dropdown menu at the top right corner of the cBioPortal home page (where your username is displayed). Your token will be downloaded as file with the following format:
```

Comment:
An image would be valuable here. It might be helpful to also mention that when token authentication is disabled (or for users who are prohibited from access) the menu option will not be present (a contrasting image).